### PR TITLE
Update cypress to 10.3.1

### PR DESCRIPTION
**Pull request by NamibiaTorres**
## Summary

We need to update Cypress to be up to date with CircleCI-Docker. At the moment they are on different versions and MyMove needs to catch up.

### Changes made for the upgrade:
* import plugins/index.js to cypress.config.js and use code in SetupNodeEvents() func
* replace `cypress/support/index.js` with `cypress/support/e2e.js`
* rename cypress/integration to cypress/e2e and  append all files in associated folders with `cy.js` instead of `.js`

## Verification Steps for Author

These are to be checked by the author.

- [x] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [x] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

Check to make sure none of our tests are breaking as a result of this dependency update.

### CODE REVIEW

File: cypress/Dockerfile.cypress [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM milmove/circleci-docker:milmove-cypress-f678d1c6a5e2085f733f5408b7affd628f9f3576
+FROM milmove/circleci-docker:milmove-cypress-fcd147c8b7ccddc93df93f4f5a2817686b4d9b5b
```
Comment (felipe-lee) [inline cypress/Dockerfile.cypress:1]: **todo**: There's one other place to update this hash:

https://github.com/transcom/mymove/blob/c95837477557cff6d06b38049f0114e5e8c120cf/.circleci/config.yml#L19

**Comment by felipe-lee** [comment]
I think the cypress tests are failing rn because there's another `cypress.json` file in the `cypress` directory. You'll probably have to remove it, and update the `cypress/Dockerfile.cypress` file to copy over the new config file instead of the old one.

https://github.com/transcom/mymove/blob/d4bda13418609a23b1c66a7f381b381f63a94784/cypress/Dockerfile.cypress#L7-L12

I haven't gotten to the mocha file so not sure exactly what that will look like with the new setup.

**Comment by rileyorr** [comment]
Hi @NamibiaTorres! I noticed there's a review request for the Truss-Design team but this PR is from July 2022. Does this PR still need a review?

**Comment by ahobson** [comment]
@rileyorr @NamibiaTorres We can definitely close this one!

**Pull request closed without merging**